lib/budget.ex

```diff
@@ -36,6 +36,7 @@ defmodule Budget do
   defp parse_to_number(string) do
     # Using String.to_float since you want to raise in case of failures
     string
+    |> String.trim_trailing("\r")
```

Comment:
I would suggest stripping the whole line before processing. Otherwise, if you add a new column, you need to move the location of that call. Call `String.trim` during or after parsing. :)